yang/frr-mgmd.yang

```diff
@@ -1,8 +1,8 @@
-module frr-igmp {
+module frr-mgmd {
```

Comment:
Can you please split off the IGMP changes into a separate PR? The PIM changes are simpler and can be merged earlier / should be merged ASAP.

Reply:
Thanks David. 
We have removed IGMP/MGMD changes from this commit. We will raise a new PR for that.